common/server_config.py

```diff
@@ -28,9 +28,21 @@ class ServerConfig(BaseModel):
     # How long to wait before considering a client as disconnected.
     client_timeout_seconds: float = 2.0
 
-    # Controls the game speed (in frames per second). A lower speed could be
-    # useful for debugging purpose.
+    # Controls the actual game speed (in frames per second).
+    # This value determines how fast the game runs in real-time.
+    # - Higher values make the game run faster than normal (e.g., 120 = 2x speed)
+    # - Lower values make the game run slower than normal (e.g., 30 = 0.5x speed)
+    # - Lower speeds can be useful for debugging purposes
     tick_rate: int = 60
+    
+    # The standard tick rate used as a reference for game timing calculations.
+    # This value ensures consistent game duration regardless of the actual tick_rate.
+    # - The game's internal time calculations are always based on this reference value
+    # - When tick_rate equals reference_tick_rate, the game runs at normal speed
+    # - When tick_rate is higher/lower than reference_tick_rate, the game runs faster/slower
+    # - In grading mode, tick_rate is set to a very high value while reference_tick_rate stays the same,
+    #   causing the game to run much faster while maintaining consistent game logic
+    reference_tick_rate: int = 60
```

Comment:
This shouldn't be a config if it's not meant to be changed. Put it under common/constants.py if it's needed on both, the server and client side?